### ISSUE TITLE
Document outline misses heading levels

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/extension-workshop#157]
See https://validator.w3.org/nu/?showoutline=yes&doc=https%3A%2F%2Fextensionworkshop.com%2F

See also https://www.w3.org/wiki/HTML/Usage/Headings/Missing for additional context.

![Showing_results_for_https___extensionworkshop_com__-_Nu_Html_Checker](https://user-images.githubusercontent.com/1514/55548551-9df5eb00-56cb-11e9-8af3-3b69aaffd92a.jpg)

**Comment by Phylis05** [comment]
Hey @muffinresearch. Still working on this? I would like to take up the issue

**Comment by muffinresearch** [comment]
@Phylis05 it's not assigned so no-one is working on it :). 

Feel free to take a look and see what needs to change to reflect the levels correctly. Let me know if you have any questions.

**Comment by muskangupta-iitr** [comment]
@muffinresearch I am Outreachy applicant and willing to contribute to mozilla. As this issue is been inactive for a while, Can I take up this issue ? and can you help me with issue by some more information as what we need to exactly do?

**Comment by muffinresearch** [comment]
@muskangupta-iitr  hi, let me check with the person this was originally assigned to first and we can go from there.

@Phylis05 how are you getting on with this issue? If you aren't able to continue working on this just let us know and we can re-assign this, also if you need a bit more time that's also fine.

**Comment by muskangupta-iitr** [comment]
@muffinresearch No worries, please assign it to me in case no one is working on it. I would be glad to work on this issue.
Thanks a lot :)
